src/Wpf/Prism.Wpf/Modularity/IModuleCatalogExtensions.cs

```diff
@@ -1,16 +1,19 @@
-﻿using System;
+using System;
 using Prism.Properties;
 
 namespace Prism.Modularity
 {
+    /// <summary>
+    /// <see cref="IModuleCatalog"/> extensions.
+    /// </summary>
     public static class IModuleCatalogExtensions
     {
         /// <summary>
-        /// Adds the module.
+        /// Adds the module to the <see cref="IModuleCatalog"/>.
         /// </summary>
-        /// <param name="catalog">Catalog</param>
-        /// <param name="mode"><see cref="InitializationMode"/></param>
-        /// <param name="dependsOn">Collection of module names (<see cref="ModuleInfo.ModuleName"/>) of the modules on which the module to be added logically depends on.</param>
+        /// <param name="catalog">The catalog to add the module to.</param>
+        /// <param name="mode">Stage on which the module to be added will be initialized.</param>
```

Comment:
the updated mode description is really incorrect. It would be the `<see cref="InitializationMode"/>`